types/office-js-preview/index.d.ts

```diff
@@ -74649,34 +74315,34 @@ declare namespace Word {
      * The insertion location types
      *
      * [Api set: WordApi]
+     * @remarks
+     * For an API call
```

Comment:
The first two lines of this description are kind of cryptic, and use pseudo-code placeholders like `insertSomething`.  How about: "Specifies the location, relative to an object, where new content is added with various insertion methods."